tools/GATK-ApplyRecalibration.cwl

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+
+hints:
+- $import: envvar-global.yml
+- $import: GATK-docker.yml
+
+inputs:
+  multithreading_nt:
+    type: int
+    default: 4
+    inputBinding:
+      position: 12
+      prefix: --nt
```

Comment:
Does GATK accept the prefixed options in any order? Most tools do, so if that is true then just specify the `prefix` to `inputBinding` and drop the `position` for clarity.